### DISCUSSION TITLE
add "RESAMPLE" to Upscale images to increase the resolution

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1830] [General] [answered]
Hello i have sent you a DONATION €20,00 over PAYPAL
about IMAGEGLASS 9 that is a beautiful program!

I have a little request:
Can you add "RESAMPLE"? Scope: First I need to "CROP" an image, then "RESAMPLE" the width and height to a larger value if possible with BicubicSharper/Bilinear/Nearest like Photoshop does to increase Image resolution?
Thanks
![image](https://github.com/d2phap/ImageGlass/assets/43314459/526dc28e-7654-4897-8f6c-021f17f433c5)

**Comment by donatopace** [comment]
Any news? Thanks.

**Reply by d2phap** [reply]
Hi @donatopace. Thanks for the donation! (I actually received 10 USD though)
About your request, as I understand it, this is something like [resizing the image #1458](https://github.com/d2phap/ImageGlass/issues/1458)(?). Also, ImageGlass mainly focuses on viewing features, editing features are not prioritized simply because I don't have enough image processing knowledge to develop it.

**Reply by d2phap** [reply, marked as answer]
Implemented in [ImageGlass 9.2 Beta](https://github.com/d2phap/ImageGlass/releases/tag/9.1.8.1104), refer to https://github.com/d2phap/ImageGlass/issues/1458#issuecomment-2454895435 for the details!
